### PR TITLE
Added send message to a channel functionality

### PR DESCRIPTION
**Pull request by daveRanjan**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -42,4 +43,9 @@ export class AppComponent {
   clearMessages() {
     this.messages="";
   }
+
+  onSend() {
+    console.log("Sending message ", this.message," to channel ", this.messageChannelName);
+    this.socketService?.send(this.messageChannelName, this.message)
```
Comment (monis01) [inline src/app/app.component.ts:49]: Is there any chance that stockService will not be available when we will be calling `onSend` ?? so do we require 
this.socketService'?'.send..... ???

**Review by monis01 — CHANGES_REQUESTED** [review]
Please check comment

File: src/app/app.component.ts (hunk 1 above)
Comment (daveRanjan) [reply src/app/app.component.ts:49]: If the user tries to send message before connecting it will break

Comment (monis01) [reply src/app/app.component.ts:49]: But the initiation of sending ba message is by clicking on button.
Once we click on button, event will triggered and at that point service will be initiated.
Or am I missing any context here?

Comment (daveRanjan) [reply src/app/app.component.ts:49]: but the user can choose to click on send button even without connecting
though this should be added as validation check. but for now this is fine.

**Pull request merged**